specification/dimensions/usagetype.md

```diff
@@ -0,0 +1,42 @@
+# Usage Type
+
+UsageType is a categorization parameter that defines how a resource or service was consumed during a specified period. The UsageType can influence the price point and also indicate whether purchased commitments are used or unused. This classification offers a nuanced view into the resource or service's consumption patterns and helps in understanding how you're billed, especially when considered alongside attributes like ChargeType and PurchaseType.
+
+The UsageType column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the
+allowed values.
+
+See [Appendix: Usage Type](#usagetype-1) for details about allowed values and governance criteria for this dimension.
+
+## Column ID
+
+UsageType
+
+## Display Name
+
+Usage Type
+
+## Description
+
+Indicates whether the record pertains to an on-demand or recurring charge, or relates to the cost or capacity using a committed discount rate or from unutilized commitments.
+
+## Content Constraints
+
+| Constraint      | Value                                    |
+| :-------------- | :--------------------------------------- |
+| Column required | True                                     |
+| Data type       | String                                   |
+| Allows nulls    | False                                    |
+| Value format    | list-of-values                           |
+
+Allowed values:
+
+| Value      | Description                                                                                                                                                                   |
+|:-----------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| On-Demand         | This usage type describes the consumption of resources or services that are billed at the standard on-demand unit price.                                                |
+| Interruptible     | The interruptible usage type pertains to resources that are provisioned using spot availability. These resources are billed at a fluctuating spot price, which is typically lower than on-demand prices.                                                                                  |
```

Comment:
Do we want to avoid the word "spot", since it's a product name?

Reply:
I would move this to supporting content.

Reply:
Agreed

Reply:
```suggestion
| Interruptible     | Interruptible usage type pertains to consumption of resources provisioned with a cloud provider's ability to stop or interrupt workloads at any time.                                                                       |
```

Reply:
Removed language referring to dynamic pricing, as it's not guaranteed to be true moving forward.

Reply:
Keeping this comment active until the PricingModel PR comes out. Will update here.
/cc @rodneyjoyce for PricingModel

Reply:
Yes, we do not refer to Spot as it is a Vendor term. We use interruptible (not a Proper Noun) in Pricing Model

Reply:
FYI - Here's the PricingModel PR. Please move any comments over there!
See #161